src/solvers/strings/string_refinement.cpp

```diff
@@ -982,13 +982,13 @@ static optionalt<exprt> get_valid_array_size(
     return {};
   }
 
-  auto n_opt = numeric_cast<std::size_t>(size_val);
-  if(!n_opt)
+  auto num_size_opt = numeric_cast<std::size_t>(size_val);
```

Comment:
`num_size_opt` is also not a great name (what does "num" stand for?). `size_opt` could be good enough?

Reply:
Got rid of that change